wger/exercises/views/exercises.py

```diff
@@ -157,6 +160,59 @@ def view(request, id, slug=None):
     return render(request, 'exercise/view.html', template_data)
 
 
+class ExerciseForm(ModelForm):
+    # Redefine some fields here to set some properties
+    # (some of this could be done with a crispy form helper and would be
+    # a cleaner solution)
+    category = ModelChoiceField(queryset=ExerciseCategory.objects.all(),
+                                widget=Select())
+    muscles = ModelMultipleChoiceField(queryset=Muscle.objects.all(),
+                                       widget=CheckboxSelectMultiple(),
+                                       required=False)
+
+    muscles_secondary = ModelMultipleChoiceField(queryset=Muscle.objects.all(),
+                                                 widget=CheckboxSelectMultiple(),
+                                                 required=False)
+
+    description = CharField(label=_('Description'),
+                            widget=Textarea,
+                            required=False)
+
+    class Meta:
+        model = Exercise
+        widgets = {'equipment': TranslatedSelectMultiple()}
+        fields = ['name_original',
+                  'category',
+                  'description',
+                  'muscles',
+                  'muscles_secondary',
+                  'equipment',
+                  'license',
+                  'license_author']
+
+    class Media:
+        js = (settings.STATIC_URL + 'yarn/tinymce/tinymce.min.js',)
+
+    def clean_name_original(self):
+        """
+        Throws a validation error if the submitted name is too similar to an existing
+        exercise's name
+        """
+        name_original = self.cleaned_data['name_original']
+        languages = load_item_languages(LanguageConfig.SHOW_ITEM_EXERCISES)
```

Comment:
You don't need to load the languages from config, here it's enough if you just filter by the one fom the current exercise. The LanguageConfig is meant so that if e.g. Russian has only a handful of exercises and ingredients, you can configure the application to also show exercises in English so that it's a bit more useful. In our case here, it's perfectly OK that there exist different exercises with the same name but in different languages